go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -1028,6 +1028,29 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 
 	// Complete gateway initialization
 	if config.OvnKubeNode.Mode == types.NodeModeDPUHost {
+		if config.Gateway.Interface == "from-pci-address" {
```

Comment:
As discussed on the call with Alin and Igal. 
- We need "from-pci-address" as a const.
- Since ""from-pci-address" is the API to enable North-South acceleration of flows through the gateway, it is ok for the API to look like this for now. With Alin we discussed consolidating all the DPU based knobs/parameters into a single parameter; however this refactor would be huge.

For now, let's document this "from-pci-address" API change. Igal will need to add this API into Cluster Network Operator.

Please also make "from-pci-address" a const variable as well.